ember-server/jvm/src/test/scala/org/http4s/ember/server/EmberServerWebSocketSuite.scala

```diff
@@ -30,18 +40,45 @@ import java.net.URI
 import java.nio.ByteBuffer
 import java.nio.charset.StandardCharsets
 
-trait EmberServerWebSocketSuitePlatform { self: EmberServerWebSocketSuite =>
+class EmberServerWebSocketSuite extends Http4sSuite with DispatcherIOFixture {
 
-  def serverURI(server: Server, path: String): URI =
-    URI.create(s"ws://${server.address}/$path")
+  def service[F[_]](wsBuilder: WebSocketBuilder[F])(implicit F: Async[F]): HttpApp[F] = {
+    val dsl = new Http4sDsl[F] {}
+    import dsl._
 
-  class Client(
-      val waitOpen: Deferred[IO, Option[Throwable]],
-      val waitClose: Deferred[IO, Option[Throwable]],
-      val messages: Queue[IO, String],
-      val pongs: Queue[IO, String],
-      val remoteClosed: Deferred[IO, Unit],
-      val client: WebSocketClient,
+    HttpRoutes
+      .of[F] {
+        case GET -> Root =>
+          Ok("Hello!")
+        case GET -> Root / "ws-echo" =>
+          val sendReceive: Pipe[F, WebSocketFrame, WebSocketFrame] = _.flatMap {
+            case WebSocketFrame.Text(text, _) => Stream(WebSocketFrame.Text(text))
+            case _ => Stream(WebSocketFrame.Text("unknown"))
+          }
+          wsBuilder.build(sendReceive)
+        case GET -> Root / "ws-close" =>
+          val send = Stream(WebSocketFrame.Text("foo"))
+          wsBuilder.build(send, _.void)
+      }
+      .orNotFound
+  }
+
+  val serverResource: Resource[IO, Server] =
+    EmberServerBuilder
+      .default[IO]
+      .withPort(port"0")
+      .withHttpWebSocketApp(service[IO])
+      .build
+
+  def fixture = (ResourceFixture(serverResource), dispatcher).mapN(FunFixture.map2(_, _))
+
+  sealed case class Client(
```

Comment:
Is `sealed` goes from Scalafix?

Reply:
Yes?

Reply:
Probably this was discussed already (remembering something), but `sealed` for `case class` just strikes the eye. It'd be nice if Scalafix could be configured to skip the rule for tests.

Reply:
Yeah, it was scalafix in https://github.com/http4s/http4s/commit/ab458086352c18cdebd5a3345423cdc829f2b414#diff-4fc59bb173a3f7944063507512a3625fd3e6baa625b5f9544c149d13cf942c4c.